core/meta/CMakeLists.txt

```diff
@@ -87,7 +87,9 @@ ROOT_OBJECT_LIBRARY(Meta
   src/TVirtualStreamerInfo.cxx
 )
 
-target_include_directories(Meta PRIVATE res ../foundation/res)
+target_include_directories(Meta PRIVATE ${CMAKE_SOURCE_DIR}/core/meta/res)
```

Comment:
Relative paths, please.

Reply:
I don't understand why it should be relative here?

Reply:
I saw you comment below, is it the case only for Core I hope?